lib/jsdom/living/nodes/HTMLImageElement-impl.js

```diff
@@ -29,10 +64,10 @@ class HTMLImageElementImpl extends HTMLElementImpl {
   }
 
   get height() {
-    if (this.hasAttribute("height")) {
-      return conversions["unsigned long"](this.getAttribute("height"));
-    }
-    return 0;
+    // Just like on browsers, if no width / height is defined, we fall back on the
+    // dimensions of the internal image data.
+    return this.hasAttribute("height") ?
```

Comment:
So, we haven't put the proper unsigned long infrastructure in yet (/cc @Sebmaster). So the way to do this is, I am pretty sure:

```js
const parsed = parseInt(this.getAttribute("height"));
return isNaN(parsed) || parsed < 0 || parsed > 2147483647 ? this.naturalHeight : parsed;
```